erc/ERCS/erc-7562.md

```diff
@@ -230,63 +330,64 @@ This means that a `Paymaster` and `Factory` contracts cannot practically be an "
 * **[SREP-040]** An `OK` staked entity is unlimited by the reputation rule.
     * Allowed in unlimited numbers in the mempool.
     * Allowed in unlimited numbers in a bundle.
-* **[SREP-050]** If a staked entity fails the second validation or fails bundle creation, its `opsSeen` is incremented by `10000`, causing it to be `BANNED`.
+* **[SREP-050]** If a staked entity fails the second validation or fails bundle creation, its `opsSeen` set to `10000`, and `opsIncluded` to zero, causing it to be `BANNED`.
```

Comment:
We realized that setting opsIncluded to zero is too harsh - requiring opsSeen to go all the way down to zero.  I think @shahafn said he'll work on a better way to trigger banning for an expected duration.  This was in the context of UREP-030 but whatever we do in UREP-030 should apply here as well.

Reply:
I suggested something else. There are three statuses for an entity: ok, throttled, banned. The opsAllowed calculation formula is only meant for unstaked entities that are OK. Once the entity is throttled or banned, the opsAllowed calcuation is much simpler and doesn't require UREP-030.

Reply:
This comment is about SREP, not UREP.  We need a way to mark something as BANNED for a certain duration (e.g. 24 hours).  We don't have statuses, just opsSeen and opsIncluded.  Therefore we need to calculate values for these two numbers, that will keep the entity banned for that duration until the decay function brings it back to THROTTLED and then OK if it behaves well.

Reply:
setting opsSeen to 10000 means it gets un-banned after ~70 hours, or 3 days.